toolchain/check/testdata/basics/raw_sem_ir/builtins.carbon

```diff
@@ -2,23 +2,36 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// ARGS: --include-diagnostic-kind compile --no-prelude-import --phase=check --dump-raw-sem-ir --builtin-sem-ir %s
```

Comment:
Why get rid of `--no-prelude-import`. Seems like the test changes as a result.

Reply:
Fixed